src/test_l1_tvSettings.c

```diff
@@ -12716,9 +12685,13 @@ void test_l1_tvSettings_positive_GetSupportedComponentColor (void)
 
 	/* Step 01: Calling tvsettings initialization and expecting the API to return success */
 	result = TvInit();
-	UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
+	UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);	
+
+	/* Step 02: Calling tvsettings to Set the SetCMSState for value true and the API to return success */
+	result = SetCMSState(true);
+	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_NONE);
```

Comment:
I think this SaveCMSState(true) has to be for SetCurrentComponentSaturation and not for GetCurrentComponentSaturation

Reply:
Moved to SetCurrentComponentSaturation